ls_1/index.html

```diff
@@ -0,0 +1,529 @@
+            <img src="/images/nice-logo.svg" alt="Nice" width="97" />
+            <img src="/images/swishfund-logo.svg" alt="Swishfund" width="135" />
+            <img src="/images/brico-logo.svg" alt="Brico" width="91" />
+            <img src="/images/praxis-logo.svg" alt="Praxis" width="101" />
+          </div>
+        </article>
+        <img src="/images/boy-learning.svg" alt="boy-picture" width="403" />
+      </section>
+      <section class="choose">
+        <h2>Shy shoose us?<span> _____</span></h2>
+        <article>
+          <img src="/images/certificate.svg" alt="Certificate" width="56" />
+          <h3>Certificate</h3>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod.A bank is a financial institution accepts deposits from
+            public and creates credit.
+          </p>
+        </article>
+        <article>
+          <img src="/images/protractor.svg" alt="Protractor" width="56" />
+          <h3>Certificate</h3>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod.A bank is a financial institution accepts deposits from
+            public and creates credit.
+          </p>
+        </article>
+        <article>
+          <img src="/images/microscope.svg" alt="Microscope" width="56" />
+          <h3>Certificate</h3>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod.A bank is a financial institution accepts deposits from
+            public and creates credit.
+          </p>
+        </article>
+        <article>
+          <img src="/images/book.svg" alt="Book" width="56" />
+          <h3>Certificate</h3>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod.A bank is a financial institution accepts deposits from
+            public and creates credit.
+          </p>
+        </article>
+        <aside class="apply">
+          <img
+            src="/images/choose-img.svg"
+            width="325"
+            alt="Why choose us image"
+          />
+          <div class="apply__details">
+            <h2>Apply for admission</h2>
+            <hr />
+            <p>Name</p>
+            <hr />
+            <p>Email</p>
+            <hr />
+            <p>Phone</p>
+            <hr />
+            <button>Explore</button>
+          </div>
+        </aside>
+      </section>
+      <a name="course"></a>
+      <section class="courses">
+        <span>MOST POPULAR COURSES</span>
+        <h2>2000+ Online courses</h2>
+        <div class="courses__slider">
+          <img src="/images/slide-left.svg" alt="Slide left" width="24" />
+          <div class="slider__img">
+            <img src="/images/slider-img1.svg" alt="Slide 1" width="270" />
+            <img
+              class="slider__img--small"
```

Comment:
What is -- notation? If you want to use it as Modifier, you shoud use _ notation, like _slider__img_small_

Reply:
Understood

Reply:
Done